apps/backend/src/modules/users/user.repository.ts

```diff
@@ -8,20 +9,29 @@ import { type UserDetailsModel } from "./user-details.model.js";
 import { type UserTaskDaysModel } from "./user-task-days.model.js";
 
 class UserRepository implements Repository {
+	private fileModel: typeof FileModel;
+
 	private userDetailsModel: typeof UserDetailsModel;
 
 	private userModel: typeof UserModel;
 
 	private userTaskDaysModel: typeof UserTaskDaysModel;
 
-	public constructor(
-		userModel: typeof UserModel,
-		userDetailsModel: typeof UserDetailsModel,
-		userTaskDaysModel: typeof UserTaskDaysModel,
-	) {
+	public constructor({
+		fileModel,
+		userDetailsModel,
+		userModel,
+		userTaskDaysModel,
+	}: {
+		fileModel: typeof FileModel;
```

Comment:
We don't want to have fileModel in user repository

Reply:
@fshabanov So what is the best way to do it? Using `fileRepository` inside `userRepository`? Or should I change the return types of the repository methods and combine them in UserEntity inside `userService`? Or should I change the queries so that the avatar is also received in `userDetailsModel`?

Reply:
Inject fileService into userService